doc/reference/configuration/cfg_audit.rst

```diff
@@ -0,0 +1,289 @@
+
+    **Example: Sending to a pipeline**
+
+    ..  code-block:: lua
+
+        box.cfg{audit_log = '| cronolog audit_tarantool.log'}
+        -- or
+        box.cfg{audit_log = 'pipe: cronolog audit_tarantool.log'}'
+
+    This starts the `cronolog <https://linux.die.net/man/1/cronolog>`_ program when the server starts
+    and sends all ``audit_log`` messages to cronolog's standard input (``stdin``).
+    If the ``audit_log`` string starts with '|' or contains the prefix ``pipe:``,
+    the string is interpreted as a Unix `pipeline <https://en.wikipedia.org/wiki/Pipeline_%28Unix%29>`_.
+
+    If log is a program, check out its pid and send it a signal to rotate logs.
+
+    **Example: Writing to a system log**
+
+    ..  warning::
+
+        Below is an example of writing audit logs to a directory shared with the system logs.
+        Tarantool allows this option, but it is **not recommended** to do this to avoid difficulties
+        when working with audit logs. System and audit logs should be written separately.
+        To do this, create separate paths and specify them.
+
+    This sample configuration sends the audit log to syslog:
+
+    ..  code-block:: lua
+
+        box.cfg{audit_log = 'syslog:identity=tarantool'}
+        -- or
+        box.cfg{audit_log = 'syslog:facility=user'}
+        -- or
+        box.cfg{audit_log = 'syslog:identity=tarantool,facility=user'}
+        -- or
+        box.cfg{audit_log = 'syslog:server=unix:/dev/log'}
+
+    If the ``audit_log`` string starts with "syslog:",
+    it is interpreted as a message for the `syslogd <https://datatracker.ietf.org/doc/html/rfc5424>`_ program,
+    which normally runs in the background of any Unix-like platform.
+    The setting can be 'syslog:', 'syslog:facility=...', 'syslog:identity=...', 'syslog:server=...' or a combination.
+
+    The ``syslog:identity`` setting is an arbitrary string that is placed at the beginning of all messages.
+    The default value is ``tarantool``.
+
+    The ``syslog:facility`` setting is currently ignored, but will be used in the future.
+    The value must be one of the `syslog <https://en.wikipedia.org/wiki/Syslog>`_ keywords
+    that tell ``syslogd`` where to send the message.
+    The possible values are ``auth``, ``authpriv``, ``cron``, ``daemon``, ``ftp``,
+    ``kern``, ``lpr``, ``mail``, ``news``, ``security``, ``syslog``, ``user``, ``uucp``,
+    ``local0``, ``local1``, ``local2``, ``local3``, ``local4``, ``local5``, ``local6``, ``local7``.
+    The default value is ``local7``.
+
+    The ``syslog:server`` setting is the locator for the syslog server.
+    It can be a Unix socket path starting with "unix:" or an ipv4 port number.
+    The default socket value is ``/dev/log`` (on Linux) or ``/var/run/syslog`` (on Mac OS).
+    The default port value is 514, which is the UDP port.
+
+    An example of a Tarantool audit log entry in the syslog:
+
+    ..  code-block:: json
```

Comment:
Please, see [the comment](https://github.com/tarantool/doc/pull/3983#discussion_r1480229836) from @Totktonada in the PR for the `audit_log` section in 3.0.  The comment referenced exactly the same example as you have below.

After that comment, I updated this example in the YAML config reference, you can check it in the [audit_log.syslog_server](https://www.tarantool.io/en/doc/latest/reference/configuration/configuration_reference/#configuration-reference-audit-syslog-server) option.

Note: the new example contains some [fields](https://docs.d.tarantool.io/en/doc/audit-log-cfg-params/enterprise/audit_log/#structure-of-audit-log-event) introduced in 3.0 - `uuid` and `severity`.

Reply:
Took your 1-line json example. This branch will be merged into latest, so it's ok to have 3.0 fields in examples.